Tests/Behat/NeosTrait.php

```diff
@@ -27,6 +29,14 @@ protected function getContext()
             /** @var \TYPO3\Neos\Domain\Repository\SiteRepository $siteRepository */
             $siteRepository = $this->objectManager->get(\TYPO3\Neos\Domain\Repository\SiteRepository::class);
 
+            /** @var WorkspaceRepository $workspaceRepository */
+            $workspaceRepository = $this->objectManager->get(WorkspaceRepository::class);
+            $liveWorkspace = $workspaceRepository->findByIdentifier('live');
```

Comment:
This does not really solve the problem. I still get the same error in the Behat tests as without this patch. If I add some debugging here, I can see that there is a "live" workspace at this point so this code does nothing new and the error remains.

Error in Behat tests of DAZ project: Features/NewsletterRendering.feature:8

Reply:
For me this fixed the issue. I'll look into this further now… Could it be that it runs further now and breaks with another workspace?

Reply:
Whatever… It consistently fails for me as well now.